vision/detect_crop_hints.rb

```diff
@@ -36,41 +40,45 @@ def detect_crop_hints project_id:, image_path:
 # This method is a duplicate of the above method, but with a different
 # description of the 'image_path' variable, demonstrating the gs://bucket/file
 # GCS storage URI format.
-def detect_crop_hints_gcs project_id:, image_path:
+def detect_crop_hints_gcs image_path:
   # [START vision_crop_hint_detection_gcs]
-  # project_id = "Your Google Cloud project ID"
   # image_path = "Google Cloud Storage URI, eg. 'gs://my-bucket/image.png'"
-  
+
   require "google/cloud/vision"
 
-  vision = Google::Cloud::Vision.new project: project_id
-  image  = vision.image image_path
+  image_annotator = Google::Cloud::Vision::ImageAnnotator.new
 
-  crop_hints = image.crop_hints
+  response = image_annotator.crop_hints_detection(
+    image: image_path
+  )
 
-  crop_hints.each do |crop_hint|
+  response.responses.each do |res|
     puts "Crop hint bounds:"
-    crop_hint.bounds.each do |bound|
-      puts "#{bound.x}, #{bound.y}"
+    res.crop_hints_annotation.crop_hints.each do |crop_hint|
+      crop_hint.bounding_poly.vertices.each do |bound|
+        puts "#{bound.x}, #{bound.y}"
+      end
     end
   end
   # [END vision_crop_hint_detection_gcs]
 end
 
 if __FILE__ == $PROGRAM_NAME
   image_path = ARGV.shift
-  project_id = ENV["GOOGLE_CLOUD_PROJECT"]
 
-  if image_path
-    detect_crop_hints image_path: image_path, project_id: project_id
-  else
-    puts <<-usage
-Usage: ruby detect_crop_hints.rb [image file path]
+  unless image_path
+    return puts <<-USAGE
```

Comment:
If we're going to indent the string in the code, use a "squiggly heredoc" so the string itself doesn't get indented. i.e. `puts <<~USAGE` ... use a tilde instead of a hyphen. Note this is supported in Ruby 2.3 or later, so beware of any tests that may be running earlier Ruby versions. If you're not sure, I suggest keeping the string outdented.